code/datums/supplypacks/robotics.dm

```diff
@@ -60,6 +60,14 @@
 	containername = "Robolimb blueprints (Morpheus)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/cybersolutions
+	name = "Cyber Solutions robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/cybersolutions)
+	cost = 20
```

Comment:
These prices are all going to need to be discussed.

Reply:
These are just equal to Morpheus given their consistently low quality.